ibis/omniscidb/tests/test_client.py

```diff
@@ -287,3 +310,24 @@ def test_cpu_execution_type(
 
     for mocked_method in mocked_methods:
         mocked_method.stop()
+
+
+@pytest.mark.parametrize(
+    'method,data',
+    [
+        ('rows', df_salary),
+        ('columnar', df_salary),
+        ('infer', df_salary),
+        ('infer', arrow_salary),
+        ('arrow', arrow_salary),
+    ],
+)
+def test_load_data(con, temp_table, method, data):
+    con.create_table(temp_table, schema=sch_salary)
+    con.load_data(temp_table, data, method=method)
+    result = con.table(temp_table).execute()
+
+    if not isinstance(data, pd.DataFrame):
+        data = data.to_pandas()  # pyarrow.Table
```

Comment:
I don't understand this, if you always want the pandas, we not simply use `assert_frame_equal(result, df_salary)` below instead?

Reply:
yes, as we are using the same data for both, it works. thanks